pkg/after_dhtnode_install.sh

```diff
@@ -10,8 +10,11 @@ error_exit()
 }
 
 if [ "$1" = "configure" ]; then
-    addgroup --system core
-    adduser --system --no-create-home dhtnode
+    addgroup --system core || true
+    adduser --system --no-create-home dhtnode || true
+
+    # Check if the user and group exist
```

Comment:
This check if user exists and it's member of core group.

Reply:
Fixed.